src/profiling.md

```diff
@@ -20,6 +20,9 @@ used successfully on Rust programs.
   domain-specific insights into parts of your code.
 - [Coz] performs *causal profiling* to measure optimization potential. It has
   Rust support via [coz-rs].
+- [flamegraph](https://github.com/flamegraph-rs/flamegraph) is a Cargo command that you can use to
+  easily profile your program. It uses `perf`/`DTrace` to profile your code and then displays the
+  results in a flame graph.
```

Comment:
This is mostly good, but I have some nits:
- Please use a trailing link rather than an inline link to match the existing links.
- Please wrap the line length at 79 to match the existing text.
- Please rephrase as "... is a Cargo command that uses `pref`/`DTrace` to profile your code ...". It's a little shorter and avoids some repetition.

Thanks!

Reply:
For some reason, a trailing link didn't work locally (it just rendered as `[flamegraph]`). I have no idea why though, the other links do work..

Reply:
It's easy to get the syntax wrong. You might have had a trailing ')' after the URL, I've done that several times.

This should work: `[flamegraph]: https://github.com/flamegraph-rs/flamegraph`

Reply:
Well, of course it works now :)